podcasts/End of Year/EndOfYear.swift

```diff
@@ -206,9 +206,13 @@ struct EndOfYear {
     }
 }
 
+extension EndOfYear {
+    static var defaultDuration = 10.seconds
```

Comment:
As we still support the 2023 version, can we switch the duration based on the `currentYear`?
```
static var defaultDuration: TimeInterval {
        switch currentYear {
        case .y2024: 10.seconds
        default: 7.seconds
        }
    }
```

Reply:
Done here: https://github.com/Automattic/pocket-casts-ios/pull/2393/commits/51402ee5e284bb2292bb1c5cad8617c06991143e